src/js/components/Tag/Tag.js

```diff
@@ -14,6 +13,7 @@ import { StyledRemoveButton, StyledTagButton } from './StyledTag';
 const Tag = forwardRef(
   ({ name, value, size, onRemove, onClick, ...rest }, ref) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
+    const RemoveIcon = theme.tag.icons.remove || FormClose;
```

Comment:
Still needs this change:
```suggestion
    const RemoveIcon = theme.tag.icons?.remove || FormClose;
```